modules/arm_plugin/README.md

```diff
@@ -0,0 +1,212 @@
+
+# [OpenVINO™ Toolkit](https://01.org/openvinotoolkit) - ARM CPU plugin
+
+OpenVINO™ ARM CPU plugin is developed in order to enable deep neural networks inference on ARM CPUs, using OpenVINO™ API. The plugin uses [ARM Compute Library\*] as a backend.
+
+## Supported Platforms
+OpenVINO™ ARM CPU plugin is supported and validated on the following platforms: 
+
+Host  | OS
+------------- | -------------
+Raspberry Pi 4 Model B   | Debian 10.3
+
+## Distribution
+OpenVINO™ ARM CPU plugin is not included into Intel® Distribution of OpenVIVO™. To use the plugin, it should be built from source code.
+
+## How to build
+### Approach #1: build OpenCV, OpenVINO and the plugin using pre-configured Dockerfile (cross-compiling)
+OpenVINO™ and ARM CPU plugin could be built in Docker container for [32-bit](Dockerfile.RPi32) and [64-bit](Dockerfile.RPi64) Debian:
+
+1. Clone `openvino_contrib` repository:
+```
+git clone --recurse-submodules --single-branch --branch=master https://github.com/openvinotoolkit/openvino_contrib.git 
+```
+2.  Go to plugin directory:
+ ```
+cd openvino_contrib/modules/arm_plugin
+```
+3. Build the plugin in Docker container:
+```
+docker image build -t arm-plugin -f Dockerfile.RPi32 .
+```
+4. Export the archive with artifacts to the current directory:
+```
+docker run -ti -v $PWD:/remote arm-plugin cp ./OV_ARM_package.tar.gz /remote
+```
+5. Extract the archive to `build` directory
+```
+mkdir build && tar -xf OV_ARM_package.tar.gz -C build
+```
+
+### Approach #2: build OpenVINO and the plugin simultaneously
+
+1. Clone `openvino` and `openvino_contrib` repositories:
+```
+git clone --recurse-submodules --single-branch --branch=master https://github.com/openvinotoolkit/openvino.git 
+git clone --recurse-submodules --single-branch --branch=master https://github.com/openvinotoolkit/openvino_contrib.git 
+```
+2. Run Docker container with mounted both `openvino` and `openvino_contrib` repositories if you do cross-compilation. If you do native compilation just skip this step:
+```
+docker run -it -v /absolute/path/to/openvino:/openvino -v /absolute/path/to/openvino_contrib:/openvino_contrib ie_cross_armhf /bin/bash 
+```
+The next commands of the procedure need to be run in `ie_cross_armhf` container.  
```

Comment:
> ie_cross_armhf

Any reference with details?

Reply:
@ilya-lavrenov do we have some description of `ie_crocc_armhf` image? I've found some details in the issue discussion https://github.com/openvinotoolkit/openvino/issues/500

Reply:
I don't know about IE docker images..

@dkurt do you know anything about this image?

Reply:
> ie_cross_armhf

This was mentioned in `build-instruction.md` file (openvino repository).
Currently this file was gone (from the master branch) or moved somewhere.
2020.3 LTS branch: https://github.com/openvinotoolkit/openvino/blob/releases/2020/3/build-instruction.md

Reply:
Hi! This is from https://github.com/openvinotoolkit/openvino/blob/releases/2021/1/build-instruction.md#cross-compilation-using-docker